policyengine_us/variables/gov/states/mi/tax/income/credits/home_heating/mi_heating_credit.py

```diff
@@ -0,0 +1,52 @@
+from policyengine_us.model_api import *
+
+
+class mi_heating_credit(Variable):
```

Comment:
this will be the final variable which will take mi_standard_allowance_heating_credit and mi_alternate_heating_credit (max)

Reply:
we need to create a mi_alternate_heating_credit_eligible.py (bool) where we check if alternate can be claimed

Reply:
mi_standard_allowance_heating_credit_eligible -> take 110% parameter and multily by fpg -> check if household_resources > or <= (true \ false) bool

Reply:
> we need to create a mi_alternate_heating_credit_eligible.py (bool) where we check if alternate can be claimed

resolved

Reply:
> we need to create a mi_standard_allowance_heating_credit_eligible.py (bool) where we check if standard allowance can be claimed

resolved

Reply:
> 

find FPG file path